test/com/esotericsoftware/kryo/KryoTestCase.java

```diff
@@ -191,8 +191,6 @@ public Input createInput (byte[] buffer) {
 		object2 = kryo.readClassAndObject(input);
 		doAssertEquals(object1, object2);
 		if (checkLength) {
-			assertEquals("Incorrect number of bytes read.", length, input.total());

```

Comment:
Why did you remove these assertions?

Reply:
Good question, I don't remember anymore, so I reverted that file and the tests still work.